src/aat/resources/features/F-037/F-037.feature

```diff
@@ -0,0 +1,68 @@
+@F-037
+Feature: F-037: Submit event for an existing case (V2)
+
+  Background: Load test data for the scenario
+    Given an appropriate test context as detailed in the test data source
+
+  @S-024
+  Scenario: should create event successfully for an existing case
+    Given a user with [an active profile in CCD]
+    And a case that has just been created as in [Standard_Full_Case_Creation_Data]
+    And a successful call [to get an event token for the case just created] as in [S-024-Prerequisite]
+    When a request is prepared with appropriate values
+    And the request [contains a case Id for just created case]
+    And the request [contains a token created as in S-024-Prerequisite]
+    And it is submitted to call the [Submit event for an existing case (V2)] operation of [CCD Data Store]
+    Then a positive response is received
+    And the response [includes the case detail for the updated case, along with a HTTP 200 OK]
+    And the response has all other details as expected
+
+  @S-022
+  Scenario: must return negative response when request does not provide valid authentication credentials
+    Given a user with [an active profile in CCD]
+    When a request is prepared with appropriate values
+    And the request [does not provide valid authentication credentials]
+    And it is submitted to call the [Submit event for an existing case (V2)] operation of [CCD Data Store]
+    Then a negative response is received
+    And the response [includes a HTTP 403 Forbidden]
```

Comment:
request for change "includes a HTTP 403 Forbidden"

Reply:
What needs to update here? it is same.

Reply:
updated